src/components/Icon/Direction.ts

```diff
@@ -0,0 +1,10 @@
+export enum IconDirection {
+  UP = "up",
+  DOWN = "down",
+  LEFT = "left",
+  RIGHT = "right",
```

Comment:
It was the other way around :)

See https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

```typescript
  Up = "UP",
  Down = "DOWN",
  Left = "LEFT",
  Right = "RIGHT",
```

Reply:
Fixed with https://github.com/Major-Eth/yearn-web-lib/pull/21/commits/f0c3c606a0ae3e1138d6b765c266de7049ab2384

Reply:
[fix: enum capitalization for values](https://github.com/Major-Eth/yearn-web-lib/pull/21/commits/f0c3c606a0ae3e1138d6b765c266de7049ab2384)